projects/storefrontlib/src/lib/ui/pages/pages.module.ts

```diff
@@ -86,6 +86,18 @@ const pageModules = [
         path: '**',
```

Comment:
Could you please add your routes above the ** route? Please add also very noticeable code comment to insert all routes above this line.